config/jobs/kubernetes/sig-scalability/sig-scalability-periodic-jobs.yaml

```diff
@@ -353,7 +353,7 @@ periodics:
       - --gcp-zone=us-east1-a
       - --ginkgo-parallel=30
       - --gke-command-group=beta
-      - --gke-create-command=container clusters create --quiet --enable-ip-alias --create-subnetwork name=ip-alias-subnet --cluster-ipv4-cidr=/11 --services-ipv4-cidr=/18
+      - --gke-create-command=container clusters create --quiet --enable-ip-alias --create-subnetwork name=ip-alias-subnet --cluster-ipv4-cidr=/11 --services-ipv4-cidr=/18 --timeout=3600
```

Comment:
Can we first start with 45 minutes (to avoid missing the regression if that happens)?

Reply:
Done. If 45m is not enough we can always increase the value.

I don't understand the regression argument though - previously we had no timeout set and sometimes it took 1h for cluster to be created. Do we have an assumption that cluster should always be created within 45?

Reply:
> I don't understand the regression argument though - previously we had no timeout set and sometimes it took 1h for cluster to be created. Do we have an assumption that cluster should always be created within 45?

The fact that we weren't able to catch the regression previously (which was the case), doesn't mean that we still shouldn't do that. It's just improvement - we will detect it (to some degree) after this change.